### PR TITLE
Add `no cover` to skip code-cov type_check and import_error

### PR DESCRIPTION
**Pull request by pankajastro**
# Description
## What is the current behavior?
Add `no cover` type_check and import_error to skip code coverage for an import statement. I'll fix it in more organized way in https://github.com/astronomer/astro-sdk/pull/1207

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: python-sdk/src/astro/utils/table.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ def find_first_table(
 
     if op_args:
         first_table = _find_first_table_from_op_args(op_args=op_args, context=context)
-    if not first_table and op_kwargs and python_callable:
+    if not first_table and op_kwargs and python_callable is not None:
```
Comment (utkarsharma2) [inline python-sdk/src/astro/utils/table.py:104]: @pankajastro Why did we add this?

Comment (pankajastro) [reply python-sdk/src/astro/utils/table.py:104]: otherwise the static check is failing since `python_callable` which is not boolean

**Review by utkarsharma2 — APPROVED** [review]

File: python-sdk/src/astro/utils/table.py (hunk 1 above)
Comment (feluelle) [reply python-sdk/src/astro/utils/table.py:104]: Interesting 🤔 🤷

Comment (pankajastro) [reply python-sdk/src/astro/utils/table.py:104]: also, I feel it does make sense if we pass empty callable 

```
val = [ ]
if val:   => won't go in this block
    print("fail") 
if val is not None:
    print("pass")
```

**Comment by pankajastro** [comment]
Merging it if any further feedback will take it separately

File: python-sdk/src/astro/utils/table.py (hunk 1 above)
Comment (feluelle) [reply python-sdk/src/astro/utils/table.py:104]:
```
if val:   => won't go in this block
    print("fail") 
```
this is expected

**Pull request merged**